src/Main.cpp

```diff
@@ -1,14 +1,77 @@
 #include<iostream>
+#include<string>
 
-#include<pjsua2.hpp>
+#include"../include/tester/ssp/SoftphoneManager.hpp"
+#include"../include/tester/ssp/SoftphoneArguments.hpp"
 
-#include<boost/asio.hpp>
 
+// void pjLibraryInit(pj::Endpoint * _endpoint)
+// {
+//     try
+//     {
+//         //Create library
+//         _endpoint->libCreate();
+
+//         // Init library
+//         pj::EpConfig ep_cfg;
+//         ep_cfg.logConfig.level = 2;
+//         _endpoint->libInit( ep_cfg );
+
+//         // Transport
+//         pj::TransportConfig tcfg;
+//         tcfg.port = 6061;
+//         _endpoint->transportCreate(PJSIP_TRANSPORT_UDP, tcfg);
+
+//         // Start library
+//         _endpoint->libStart();
+//         std::cout << "*** PJSUA2 STARTED ***" << std::endl;
+//     }
+//     catch(pj::Error & err)
+//     {
+//         std::cerr << "Exception: " << err.info() << std::endl;
+//     }
+// }
 
 int main(){
-    std::cout<<"Hi" << std::endl;
+    SoftphoneManager manager(5060, "192.168.1.46");
+
+    manager.pjLibraryInit();
```

Comment:
Allow `pjLibraryInit` to take a number as a parameter, to use for the log level in the configuration

Reply:
done